src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -30,18 +30,7 @@
   "changed": [
     {
       "what": [
-        "the visibility of the inline record editor option in a record list to hidden if a module isn't provided."
-      ],
-      "why": [
-        "to not confuse the user that the option is available and can be used."
-      ],
-      "refs": [
-        "https://github.com/cortezaproject/corteza/commit/b96ea0ea236eec6903c881529bbcbc9e393220ab"
-      ]
-    },
-    {
-      "what": [
-        "loading of page resources when switching between pages."
+        "loading of page resources when switching between pages. Previously, after switching from one page to another, the previous page's resources continued to be fetched. Currently, when the same is done, only the current page's resources are fetched."
```

Comment:
```
"loading of page resources when switching between pages.",
"Previously, after switching from one page to another, the previous page's resources continued to be fetched.",
"Currently, only the current page's resources are fetched when the same is done."
```

It needs to be like this -- each sentence should go in a separate line in this array. The same would apply to all contents in change logs.

Reply:
I like the elaboration on what you wrote initially, it clears out the ambiguity. What you could also try is to get rid of the ambiguity in the first place (reword everything to not cause questions).

I'd probably do something along the lines of (rewording what you have that is)

```
(Changed )"page navigation to stop loading previous resources in case we've navigated to a new page.",
"This change reduces the amount of needless processing since the old resources would be discarded right away."
```

You could also consider moving my second line into a `why` since it elaborates on why we've done it; the what only points it out.